sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp

```diff
@@ -139,7 +139,9 @@
 
 uint16 pvmp3_decode_huff_cw_tab0(tmp3Bits *pMainData)
 {
+#if (CC_PLATFORM != CC_PLATFORM_NX_WINDOWS)
     OSCL_UNUSED_ARG(pMainData);
+#endif
```

Comment:
Should remember that, if modify any 3rd codes, should clone it and modify it.

Reply:
ok